client/lobbies/race-picker.jsx

```diff
@@ -87,27 +88,31 @@ export const StyledRaceIcon = styled(RaceIcon)`
 export default class RacePicker extends React.Component {
   static propTypes = {
     race: PropTypes.oneOf(['r', 'p', 't', 'z']).isRequired,
+    disableRace: PropTypes.oneOf(['r', 'p', 't', 'z']),
     size: PropTypes.oneOf([RACE_PICKER_SIZE_MEDIUM, RACE_PICKER_SIZE_LARGE]),
     allowRandom: PropTypes.bool,
     onSetRace: PropTypes.func,
   }
 
   static defaultProps = {
+    disableRace: null,
```

Comment:
Whenever we get around to moving more of these components to TS, `undefined` tends to function more nicely than `null`, I'd probably just leave this out of the defaultProps if the intention is that it has no value by default